oak-store-document/pom.xml

```diff
@@ -43,8 +43,8 @@
         <configuration>
           <instructions>
             <Import-Package>
-              com.mongodb*;version="[3.8, 4)";resolution:=optional,
-              org.bson*;version="[3.8, 4)";resolution:=optional,
+              com.mongodb*;version="[5.2, 5.3)";resolution:=optional,
+              org.bson*;version="[5.2, 5.3)";resolution:=optional,
```

Comment:
we already have version `5.5.0` released for mongo-driver-sync [1]

[1] https://mvnrepository.com/artifact/org.mongodb/mongodb-driver-sync

Reply:
Yes, but we should test it before allowing it. We can do it in a new PR after this is merged, to avoid further delays on this PR.